sequel-ace.xcodeproj/project.pbxproj

```diff
@@ -4524,7 +4524,7 @@
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				COMBINE_HIDPI_IMAGES = YES;
-				CURRENT_PROJECT_VERSION = 2118;
+				CURRENT_PROJECT_VERSION = 2121;
```

Comment:
We need to bump the build number way up in case we need to release 2.3.x emergent fixes.
Let's bump to 3000 @Kaspik

Reply:
Done :)